charts/kubeai/templates/secret.yaml

```diff
@@ -1,4 +1,6 @@
-{{- if .Values.secrets.huggingface.create -}}
+# Only create the secret if the token is not empty.
+# See: https://github.com/substratusai/kubeai/issues/232
+{{- if and .Values.secrets.huggingface.create (not (empty .Values.secrets.huggingface.token)) -}}
```

Comment:
This conditional should be for the .data.value IMO, not the whole secret

Reply:
You mean remove the and?

Reply:
It's already for the data right now